### PR TITLE
Added static lib packaging

### PR DESCRIPTION
**Pull request by JohanMabille**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

**Comment by SylvainCorlay** [comment]
LGTM

File: recipe/install.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+make install
+
+if [[ "$PKG_NAME" == *static ]]
+then
+	# relying on conda to dedup package
+	echo "Keeping all files, conda will dedupe"
```
Comment (SylvainCorlay) [inline recipe/install.sh:6]: How do we know that `libnghttp2` will be the one getting the shared objects etc? Is it because it is built first?

Comment (wolfv) [reply recipe/install.sh:6]: The static package should pin exactly the non static one . Then it will be installed into host and deduped by conda build

**Review by xhochy — APPROVED** [review]

**Comment by isuruf** [comment]
This probably needs `c-ares` and `libev` to package the static libraries.

**Comment by JohanMabille** [comment]
`libev` already packages the [static library](https://github.com/conda-forge/libev-feedstock/blob/master/recipe/meta.yaml#L40)
I've opened https://github.com/conda-forge/c-ares-feedstock/pull/10 for the packaging of the static library of c-ares.

**Comment by xhochy** [comment]
@conda-forge/nghttp2 This is ready for review/merge.

**Pull request merged**